### PR TITLE
feat: update request method in android and SwiftWeb3AuthFlutterPlugin

### PR DESCRIPTION
**Pull request by grvgoel81**
## Motivation and Context
* update request methid in android and SwiftWeb3AuthFlutterPlugin.
* Example json: ChainConfig(chainId: "0x89", rpcTarget: "https://polygon-rpc.com/"),
          "eth_sendTransaction",
          [
            {
              "from": address.hexEip55,
              "to": "0x4B0897b0513FdBeEc7C469D9aF4fA6C0752aBea7",
              "value": "0x0",
              "data": "0x",
            }
          ],

Jira Link: https://toruslabs.atlassian.net/browse/PD-4089

## Description
Update request method to parse json data in params field.

## How has this been tested?
By running on emulator.

## Screenshots (if appropriate):
Screenshot 2024-10-28 at 5 22 11 PM
Screenshot 2024-10-28 at 5 23 01 PM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project. (run lint)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code requires a db migration.

### CODE REVIEW

**Comment by metalurgical** [comment]
We've seen before (torus-utils) that when resorting to doing this sort of thing, it indicates an underlying problem with generics and serialization. It would be a better idea to fix that problem instead.

File: ios/Classes/SwiftWeb3AuthFlutterPlugin.swift [hunk 1]
```diff
@@ -253,4 +253,71 @@ struct RequestJson: Codable {
...
+        let paramsArray = try container.decode([[String: AnyCodable]].self, forKey: .requestParams)
+        requestParams = paramsArray.map { paramDict in
+            // Convert each dictionary to JSON string
+            let jsonData = try! JSONEncoder().encode(paramDict)
+            return String(data: jsonData, encoding: .utf8)!
+        }
+    }
 }
+
+struct AnyCodable: Codable {
```
Comment (chaitanyapotti) [inline ios/Classes/SwiftWeb3AuthFlutterPlugin.swift:278]: which struct extends AnyCodable?

Comment (grvgoel81) [reply ios/Classes/SwiftWeb3AuthFlutterPlugin.swift]: this is removed now

**Comment by grvgoel81** [comment]
Closing this as changes added in https://github.com/Web3Auth/web3auth-flutter-sdk/pull/83

**Pull request closed without merging**